apps/backend/.env.example

```
@@ -14,6 +14,15 @@ REDIS_PASSWORD="skku1234"
 AWS_ACCESS_KEY_ID="skku"
 AWS_SECRET_ACCESS_KEY="skku1234"
 
+### RabbitMQ judge routing ###
+# TEST_KEY and REJUDGE_KEY fall back to JUDGE_SUBMISSION_ROUTING_KEY when unset,
+# preserving the former single judge request queue behavior.
+SUBMISSION_KEY="judge.submission"
+TEST_KEY="judge.test"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)